conf/application.conf

```diff
@@ -43,6 +43,7 @@ play.http.errorHandler = "controllers.ErrorHandler"
 # Config for calculator
 calculatorConfig {
     maximumLengthOfPaymentPlan = 24
+    numberOfWorkingDaysToAdd = 6
```

Comment:
`numberOfWorkingDaysToAdd` is a bit vague here without context - maybe a shorter version of something like `numberOfWorkingDaysUntilFirstPaymentCanBeTaken` is more descriptive?. `nWorkingDaysUntilFirstPaymentCanBeTaken` maybe? You might be able to think of something better though  😄